tasks/4-deploy.yaml

```diff
@@ -147,7 +147,7 @@ spec:
               yq w - vcsInfo.repoUrl "${GIT_URL}" | \
               yq w - vcsInfo.branch "${GIT_REVISION}" | \
               yq w - image.repository "${IMAGE_URL}" | \
-              yq w - image.tag "${IMAGE_VERSION}" | \
+              yq w - --style single image.tag "${IMAGE_VERSION}" | \
```

Comment:
@seansund there is another task where we have the same logic, not critical but for good measure
https://github.com/IBM/ibm-garage-tekton-tasks/blob/master/tasks/9-helm-release.yaml#L194